lib/src/internal/response_wrapper/thread_list_result_wrapper.dart

```diff
@@ -11,7 +11,7 @@ abstract class IThreadListResultWrapper {
   List<IThreadMember> get selfThreadMembers;
 
   /// Whether there are potentially additional threads that could be returned on a subsequent call
-  bool get hasMore;
+  bool? get hasMore;
```

Comment:
This is a breaking change

Reply:
null in this case is equal to false so I think if discord returned null we can assume its false

Reply:
Done